index.bs

```diff
@@ -296,7 +298,7 @@ The API is defined by the following Web IDL fragment.
 
     interface ScopedCredentialInfo {
         readonly attribute Credential           credential;
-        readonly attribute any                  publicKey;
+        readonly attribute CryptoKey            publicKey;
```

Comment:
You're assuming the RP would prefer a JsonWebKey, and that JWK will be the preference for the lifetime of the standard. Using the `CryptoKey` type gives the implementer flexibility to, today, choose to use DER-encoding of the SPKI field for that pubkey, and future-proofing that type.

I'm not convinced that it's much burden for an implementer writing authentication software to search the web for how to serialize a `CryptoKey`-type. To be honest, my reservations with this have to do with it returning a `Promise`, adding another `.then()` onto the chain of a `makeCredential()`. But IMO the future-proofing is worth that one.